src/pages/blog/videos/_video-player.js

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Flex } from 'components/containers'
+import CloseIcon from 'images/svg/close.svg'
+import device from 'themes/device'
+
+const VideoWrapper = styled.div`
+    position: fixed;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0, 0, 0, 0.9);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    z-index: 1000;
+`
+
+const StyledFlex = styled(Flex)`
+    width: 80%;
+    margin-right: -36px;
+
+    @media ${device.tablet} {
+        width: 90%;
+        margin-right: -10px;
+    }
+`
+
+const CloseButton = styled.img`
+    cursor: pointer;
+`
+
+const StyledVideoPlayer = styled.video`
+    width: 80%;
+    position: relative;
+    background-color: var(--color-black);
+
+    @media ${device.tablet} {
+        width: 90%;
+    }
+`
+
+const VideoPlayer = ({ video_src, closeVideo }) => {
+    const handleEscape = (e) => {
+        if (e.keyCode === 27) {
+            closeVideo()
+        }
```

Comment:
I think we should use the `event.key` as `event.keyCode` is marked as deprecated. https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode